manage.py

```diff
@@ -1,57 +1,149 @@
 #!/usr/bin/env python
 
-import json
+
+import tabulate
 import typer
-from typing import List
+from typing import List, Optional
 from typing_extensions import Annotated
 from sqlmodel import Session
 from sqlmodel.sql.expression import select
 from fastapi import Depends
 from fastapi.params import Security
 from app.models.users import User, UserScopes
+from app.auth.security import get_password_hash
 from app.db import get_session
 from app.main import create_app
 
+
 app = typer.Typer()
 
-session: Session = next(get_session())
+
+def init_session(typer_app: typer.Typer, session: Session) -> None:
+    typer_app.db_session = session
 
 
 @app.command()
```

Comment:
Do we need documentation and tests around scopes or is that a separate thing / done already?

Reply:
I think we already have scopes docs `https://github.com/ministryofjustice/laa-civil-case-api/blob/main/docs/source/documentation/scopes.erb`

Reply:
Added tests for user scopes now